docs/designs/restore-and-gc.md

```diff
@@ -20,20 +20,40 @@ github:
   userCache: https:///some-blob-service/github-user-cache.json
 ```
 
-The restored files will be stored at `%DOCFX_APPDATA_PATH%/downloads/{url-short-path}/{version}+{etag}` like:
+The restored files will be stored at `%DOCFX_APPDATA_PATH%/downloads/{url-short-path}/{number}` like:
 
 ```text
 `%DOCFX_APPDATA_PATH%`
     | - downloads
         | - raw.gith..tent.com+docascode+docfx-te..ndencies+62b0448+extend1.yml+dc363b0e
-            | - 18b265bc4e9a6b4fdff8b5210caf9e3541f4cb2d+%22f02fde5b4f3817266458170ba048c7fc2de46287%22
-            | - 3111eb522b092c9cef5cd0ccd86892cf847592d6+%2299a5dbb8d73da1ec452d74c0ab45b11ebe8615a0%22
+            | - 1
+            | - 2
 
 ```
 
+And there is another index file `index.txt` under `%DOCFX_APPDATA_PATH%/downloads/{url-short-path}/` tracking the detail info of each `{number} file`.
+
+```text
+`%DOCFX_APPDATA_PATH%`
+    | - downloads
+        | - raw.gith..tent.com+docascode+docfx-te..ndencies+62b0448+extend1.yml+dc363b0e
+            | - 1
+            | - 2
+            | - index.txt
+
+```
+
+`index.txt`:
+```text
+1 {version} {etag} {date} {in-use}
+2 {version} {etag} {date} {in-use}
+```
```

Comment:
can this be `JSON`? because we will add new properties to it, also makes it more readable

Reply:
I am going to change it to json when i am doing the init implementation :)